Python/Calculate-distance/Readme.md

```diff
@@ -0,0 +1,50 @@
+# Distance Calculating Django App
+App that can show distance between user's location and searched location on map.
+
+# Packages Used
```

Comment:
Please also mention which version of Django this project is compatible with.  Also add it in requirements.txt.

Reply:
@atarax665 I have made the changes, do I need to make any other changes?